app/src/main/java/com/simply/birthdayapp/presentation/ui/components/BaseButton.kt

```diff
@@ -28,20 +29,22 @@ fun AuthButton(
     enabled: Boolean = true,
     fontSize: TextUnit = 15.sp,
     textStyle: TextStyle = AppTheme.typography.bold,
+    textModifier:Modifier = Modifier,
```

Comment:
Format the code. And it's a good practice to have just one modifier as an argument. For nested child views usually we don't pass modifiers from outside. In this case for text composable, which is inside the button.

Reply:
Fixed